### PR TITLE
copr: vectorize ceilinttodec

### PR DESCRIPTION
**Pull request by AerysNan**
Signed-off-by: AerysNan <aerysnan@gmail.com>

PCP #5751

###  What have you changed?

Vectorize CeilIntToDec.

###  What is the type of the changes?

Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?

Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No.

###  Does this PR affect `tidb-ansible`?

No.

### CODE REVIEW

**Comment by AerysNan** [comment]
@sre-bot /run-integration-copr-test

**Comment by AerysNan** [comment]
@sre-bot /run-integration-copr-test

**Comment by breeswish** [comment]
@mapleFU @iosmanthus PTAL, thanks!

File: components/tidb_query/src/rpn_expr/impl_math.rs [hunk 1]
```diff
@@ -96,6 +96,18 @@ impl Ceil for CeilDecToDec {
...
 
+pub struct CeilIntToDec;
+
+impl Ceil for CeilIntToDec {
+    type Input = Int;
+    type Output = Decimal;
+
+    #[inline]
+    fn ceil(_ctx: &mut EvalContext, arg: &Self::Input) -> Result<Option<Self::Output>> {
+        Ok(Some(Self::Output::from(*arg)))
```
Comment (iosmanthus) [inline components/tidb_query/src/rpn_expr/impl_math.rs]: Please use `Decimal::from`, it's more clear.

Comment (AerysNan) [reply components/tidb_query/src/rpn_expr/impl_math.rs]: updated

**Pull request merged**